packages/core/core-flows/src/payment-collection/workflows/create-payment-session.ts

```diff
@@ -41,7 +40,17 @@ export const createPaymentSessionsWorkflow = createWorkflow(
       }
     )
 
-    const created = createPaymentSessionStep(paymentSessionInput)
+    // Note: We are deleting an existing active session before creating a new one
+    // for a payment collection as we don't support split payments at the moment.
+    // When we are ready to accept split payments, this along with other workflows
+    // need to be handled correctly
+    const [created] = parallelize(
+      createPaymentSessionStep(paymentSessionInput),
+      deletePaymentSessionStep({
+        payment_session_id: paymentCollection.payment_sessions?.[0].id,
```

Comment:
We can pass all the IDs here to delete them all.
In case we support multiple, all the sessions have to be created at the same time anyway, therefore all the existing ones can be removed.